### ISSUE TITLE
Corrupted question shows "Something's gone wrong" with no more info, and breaks "Add series" modal on the dashboard, and breaks Archive page

### ISSUE THREAD

**Issue by camsaul** [metabase/metabase#15222]
Steps to repro:

1.  Go to <horror-show>/question/1472 
2. See "Something's gone wrong" page with no more info. 
3. `Cannot read property 'type' of null` in the browser console. API response is a 200

I suspect it can't read the empty viz settings or something? We should have better error handling around this, and more graceful fallback, an maybe more info for debugging purposes. Not sure what's part of the data it doesn't like, but I think it should still be able to render the rest of the page

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by flamber** [comment]
It is caused by `dataset_query` being empty - I can only set to `""` in the database, but the request says it's `null`:

`{"description":null,"archived":false,"collection_position":null,"table_id":null,"result_metadata":null,"creator":{<removed>},"can_write":true,"database_id":3,"enable_embedding":false,"collection_id":null,"query_type":null,"name":"A","dashboard_count":0,"creator_id":3,"updated_at":"2021-03-12T11:40:05.745649Z","made_public_by_id":null,"embedding_params":null,"cache_ttl":null,"dataset_query":null,"id":1472,"display":"table","visualization_settings":{},"collection":null,"created_at":"2021-03-12T11:40:05.745649Z","public_uuid":null}`

**Comment by flamber** [comment]
Changing priority on this, since this is actually the root cause of #13465 and https://discourse.metabase.com/t/cant-add-series-to-graphs-in-dashboard/12938 and https://discourse.metabase.com/t/charts-comparision-is-not-working-after-upgrade-0-37-5/12942

----

If you experience this problem, then check `report_card.dataset_query = ''` in the application database. It might have other values, but it should be a valid JSON object.

**Comment by camsaul** [comment]
Normally I don't think you should be able to set a blank query; I think if the column is empty the normalization logic is just returning `nil` instead. Either way, if the query is invalid/blank we should handle it better

**Comment by flamber** [comment]
It can also be caused by updating a question's `visualization_settings` to the following via the API:
`{"column_settings":"{\"(keyword [\\\"name\\\" \\\"ID\\\"])\":{\"number_separators\":\".\"}}"}`

Yes, complete nonsense, but the API accepts it.
Trying to view the question with now show "Something's gone wrong" and it's impossible to update the question with a corrected vizsettings, and even trying to delete the question via API also fails.
The only solution is the manually update the application database `report_card.visualization_settings='{}'`

<details><summary>Full stacktrace</summary>

```
2021-12-22 22:23:47,756 ERROR middleware.log :: GET /api/card/494 500 3.2 ms (1 DB calls) 
{:via
 [{:type java.lang.UnsupportedOperationException,
   :message "nth not supported on this type: Character",
   :at [clojure.lang.RT nthFrom "RT.java" 992]}],
 :trace
 [[clojure.lang.RT nthFrom "RT.java" 992]
  [clojure.lang.RT nth "RT.java" 940]
  [metabase.models.interface$normalize_visualization_settings$normalize_column_settings__31384$iter__31385__31389$fn__31390
   invoke
   "interface.clj"
   113]
  [clojure.lang.LazySeq sval "LazySeq.java" 42]
  [clojure.lang.LazySeq seq "LazySeq.java" 51]
  [clojure.lang.RT seq "RT.java" 535]
  [clojure.core$seq__5419 invokeStatic "core.clj" 139]
  [clojure.core.protocols$seq_reduce invokeStatic "protocols.clj" 24]
  [clojure.core.protocols$fn__8168 invokeStatic "protocols.clj" 75]
  [clojure.core.protocols$fn__8168 invoke "protocols.clj" 75]
  [clojure.core.protocols$fn__8110$G__8105__8123 invoke "protocols.clj" 13]
  [clojure.core$reduce invokeStatic "core.clj" 6830]
  [clojure.core$into invokeStatic "core.clj" 6897]
  [clojure.core$into invoke "core.clj" 6889]
  [metabase.models.interface$normalize_visualization_settings$normalize_column_settings__31384
   invoke
   "interface.clj"
   113]
  [metabase.models.interface$normalize_visualization_settings invokeStatic "interface.clj" 127]
  [metabase.models.interface$normalize_visualization_settings invoke "interface.clj" 107]
  [clojure.core$comp$fn__5825 invoke "core.clj" 2573]
  [toucan.models$apply_type_fns$iter__20983__20987$fn__20988 invoke "models.clj" 304]
  [clojure.lang.LazySeq sval "LazySeq.java" 42]
  [clojure.lang.LazySeq seq "LazySeq.java" 51]
  [clojure.lang.Cons next "Cons.java" 39]
  [clojure.lang.RT next "RT.java" 713]
  [clojure.core$next__5403 invokeStatic "core.clj" 64]
  [clojure.core.protocols$fn__8181 invokeStatic "protocols.clj" 169]
  [clojure.core.protocols$fn__8181 invoke "protocols.clj" 124]
  [clojure.core.protocols$fn__8136$G__8131__8145 invoke "protocols.clj" 19]
  [clojure.core.protocols$seq_reduce invokeStatic "protocols.clj" 31]
  [clojure.core.protocols$fn__8168 invokeStatic "protocols.clj" 75]
  [clojure.core.protocols$fn__8168 invoke "protocols.clj" 75]
  [clojure.core.protocols$fn__8110$G__8105__8123 invoke "protocols.clj" 13]
  [clojure.core$reduce invokeStatic "core.clj" 6830]
  [clojure.core$into invokeStatic "core.clj" 6897]
  [clojure.core$into invoke "core.clj" 6889]
  [toucan.models$apply_type_fns invokeStatic "models.clj" 302]
  [toucan.models$apply_type_fns invoke "models.clj" 299]
  [toucan.models$do_post_select invokeStatic "models.clj" 349]
  [toucan.models$do_post_select invoke "models.clj" 344]
  [toucan.db$do_post_select$iter__21209__21213$fn__21214 invoke "db.clj" 373]
  [clojure.lang.LazySeq sval "LazySeq.java" 42]
  [clojure.lang.LazySeq seq "LazySeq.java" 51]
  [clojure.lang.RT seq "RT.java" 535]
  [clojure.lang.LazilyPersistentVector create "LazilyPersistentVector.java" 44]
  [clojure.core$vec invokeStatic "core.clj" 379]
  [clojure.core$vec invoke "core.clj" 369]
  [toucan.db$do_post_select invokeStatic "db.clj" 372]
  [toucan.db$do_post_select invoke "db.clj" 363]
  [toucan.db$simple_select invokeStatic "db.clj" 394]
  [toucan.db$simple_select invoke "db.clj" 383]
  [toucan.db$simple_select_one invokeStatic "db.clj" 420]
  [toucan.db$simple_select_one invoke "db.clj" 409]
  [toucan.db$select_one invokeStatic "db.clj" 627]
  [toucan.db$select_one doInvoke "db.clj" 620]
  [clojure.lang.RestFn applyTo "RestFn.java" 139]
  [clojure.lang.Var applyTo "Var.java" 705]
  [clojure.core$apply invokeStatic "core.clj" 673]
  [clojure.core$apply invoke "core.clj" 662]
  [toucan.models$invoke_model invokeStatic "models.clj" 380]
  [toucan.models$invoke_model doInvoke "models.clj" 368]
  [clojure.lang.RestFn invoke "RestFn.java" 445]
  [toucan.models$invoke_model invokeStatic "models.clj" 378]
  [toucan.models$invoke_model invoke "models.clj" 368]
  [clojure.lang.AFn applyToHelper "AFn.java" 156]
  [clojure.lang.RestFn applyTo "RestFn.java" 132]
  [clojure.core$apply invokeStatic "core.clj" 669]
  [clojure.core$apply invoke "core.clj" 662]
  [toucan.models$invoke_model_or_instance invokeStatic "models.clj" 395]
  [toucan.models$invoke_model_or_instance doInvoke "models.clj" 391]
  [clojure.lang.RestFn invoke "RestFn.java" 423]
  [metabase.models.card.CardInstance invoke "card.clj" 26]
  [metabase.api.card$fn__58204 invokeStatic "card.clj" 171]
  [metabase.api.card$fn__58204 invoke "card.clj" 168]
  [compojure.core$wrap_response$fn__32066 invoke "core.clj" 160]
  [compojure.core$wrap_route_middleware$fn__32050 invoke "core.clj" 132]
  [compojure.core$wrap_route_info$fn__32055 invoke "core.clj" 139]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 151]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 153]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078 invoke "core.clj" 200]
  [metabase.server.middleware.auth$enforce_authentication$fn__70282 invoke "auth.clj" 14]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078 invoke "core.clj" 200]
  [compojure.core$make_context$handler__32106 invoke "core.clj" 289]
  [compojure.core$make_context$fn__32110 invoke "core.clj" 299]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$make_context$fn__32110 invoke "core.clj" 300]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$make_context$fn__32110 invoke "core.clj" 300]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$make_context$fn__32110 invoke "core.clj" 300]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [metabase.api.routes$fn__79759$fn__79762 invoke "routes.clj" 56]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078 invoke "core.clj" 200]
  [clojure.lang.AFn applyToHelper "AFn.java" 160]
  [clojure.lang.AFn applyTo "AFn.java" 144]
  [clojure.core$apply invokeStatic "core.clj" 667]
  [clojure.core$apply invoke "core.clj" 662]
  [metabase.server.routes$fn__79904$fn__79905 doInvoke "routes.clj" 57]
  [clojure.lang.RestFn invoke "RestFn.java" 436]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078 invoke "core.clj" 200]
  [compojure.core$make_context$handler__32106 invoke "core.clj" 289]
  [compojure.core$make_context$fn__32110 invoke "core.clj" 299]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [compojure.core$wrap_route_matches$fn__32059 invoke "core.clj" 152]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078$f__32079$respond_SINGLEQUOTE___32080 invoke "core.clj" 197]
  [metabase.server.routes$fn__79892$fn__79894 invoke "routes.clj" 41]
  [compojure.core$routes$fn__32078$f__32079 invoke "core.clj" 198]
  [compojure.core$routes$fn__32078 invoke "core.clj" 200]
  [metabase.server.middleware.exceptions$catch_uncaught_exceptions$fn__77364 invoke "exceptions.clj" 98]
  [metabase.server.middleware.exceptions$catch_api_exceptions$fn__77361 invoke "exceptions.clj" 86]
  [metabase.server.middleware.log$log_api_call$fn__80359$fn__80360$fn__80361 invoke "log.clj" 209]
  [metabase.driver.sql_jdbc.execute.diagnostic$do_with_diagnostic_info invokeStatic "diagnostic.clj" 15]
  [metabase.driver.sql_jdbc.execute.diagnostic$do_with_diagnostic_info invoke "diagnostic.clj" 9]
  [metabase.server.middleware.log$log_api_call$fn__80359$fn__80360 invoke "log.clj" 201]
  [toucan.db$_do_with_call_counting invokeStatic "db.clj" 216]
  [toucan.db$_do_with_call_counting invoke "db.clj" 209]
  [metabase.server.middleware.log$log_api_call$fn__80359 invoke "log.clj" 200]
  [metabase.server.middleware.browser_cookie$ensure_browser_id_cookie$fn__83572 invoke "browser_cookie.clj" 30]
  [metabase.server.middleware.security$add_security_headers$fn__59226 invoke "security.clj" 147]
  [metabase.server.middleware.json$wrap_json_body$fn__82716 invoke "json.clj" 62]
  [metabase.server.middleware.json$wrap_streamed_json_response$fn__82734 invoke "json.clj" 98]
  [metabase.server.middleware.offset_paging$handle_paging$fn__59250 invoke "offset_paging.clj" 42]
  [ring.middleware.keyword_params$wrap_keyword_params$fn__83839 invoke "keyword_params.clj" 55]
  [ring.middleware.params$wrap_params$fn__83855 invoke "params.clj" 69]
  [metabase.server.middleware.misc$maybe_set_site_url$fn__34077 invoke "misc.clj" 59]
  [metabase.server.middleware.session$bind_current_user$fn__44978$fn__44979 invoke "session.clj" 257]
  [metabase.server.middleware.session$do_with_current_user invokeStatic "session.clj" 238]
  [metabase.server.middleware.session$do_with_current_user invoke "session.clj" 230]
  [metabase.server.middleware.session$bind_current_user$fn__44978 invoke "session.clj" 256]
  [metabase.server.middleware.session$wrap_current_user_info$fn__44965 invoke "session.clj" 216]
  [metabase.server.middleware.session$wrap_session_id$fn__44951 invoke "session.clj" 162]
  [metabase.server.middleware.auth$wrap_api_key$fn__70290 invoke "auth.clj" 27]
  [ring.middleware.cookies$wrap_cookies$fn__83759 invoke "cookies.clj" 216]
  [metabase.server.middleware.misc$add_content_type$fn__34060 invoke "misc.clj" 27]
  [metabase.server.middleware.misc$disable_streaming_buffering$fn__34085 invoke "misc.clj" 76]
  [ring.middleware.gzip$wrap_gzip$fn__83801 invoke "gzip.clj" 86]
  [metabase.server.middleware.misc$bind_request$fn__34088 invoke "misc.clj" 93]
  [metabase.server.middleware.ssl$redirect_to_https_middleware$fn__83588 invoke "ssl.clj" 38]
  [metabase.server$async_proxy_handler$fn__80132 invoke "server.clj" 71]
  [metabase.server.proxy$org.eclipse.jetty.server.handler.AbstractHandler$ff19274a handle nil -1]
  [org.eclipse.jetty.server.handler.HandlerWrapper handle "HandlerWrapper.java" 127]
  [org.eclipse.jetty.server.Server handle "Server.java" 516]
  [org.eclipse.jetty.server.HttpChannel lambda$handle$1 "HttpChannel.java" 388]
  [org.eclipse.jetty.server.HttpChannel dispatch "HttpChannel.java" 633]
  [org.eclipse.jetty.server.HttpChannel handle "HttpChannel.java" 380]
  [org.eclipse.jetty.server.HttpConnection onFillable "HttpConnection.java" 277]
  [org.eclipse.jetty.io.AbstractConnection$ReadCallback succeeded "AbstractConnection.java" 311]
  [org.eclipse.jetty.io.FillInterest fillable "FillInterest.java" 105]
  [org.eclipse.jetty.io.ChannelEndPoint$1 run "ChannelEndPoint.java" 104]
  [org.eclipse.jetty.util.thread.strategy.EatWhatYouKill runTask "EatWhatYouKill.java" 338]
  [org.eclipse.jetty.util.thread.strategy.EatWhatYouKill doProduce "EatWhatYouKill.java" 315]
  [org.eclipse.jetty.util.thread.strategy.EatWhatYouKill tryProduce "EatWhatYouKill.java" 173]
  [org.eclipse.jetty.util.thread.strategy.EatWhatYouKill run "EatWhatYouKill.java" 131]
  [org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread run "ReservedThreadExecutor.java" 386]
  [org.eclipse.jetty.util.thread.QueuedThreadPool runJob "QueuedThreadPool.java" 883]
  [org.eclipse.jetty.util.thread.QueuedThreadPool$Runner run "QueuedThreadPool.java" 1034]
  [java.lang.Thread run "Thread.java" 829]],
 :cause "nth not supported on this type: Character",
 :message "nth not supported on this type: Character"}
```

</details>

**Comment by paoliniluis** [comment]
Ok, so now if there's a corruption on any card it will reload the page and the add series won't work anymore

**Comment by BertrandToulas** [comment]
Following the discussions in this issue and all other related issues, I've archived or fixed all the cards that seemed to be generating errors in our server logs, but I still can't use "Add series"; the page just refreshes, as described in my original issue (#26510 ). I've also checked `report_card.dataset_query`, but there doesn't seem to be any empty query.

Short of removing access to all the questions from my user's instance, and then re-adding collections one by one until I find the culprit(s), are there any other workarounds to get rid of the issue? Anything else I should look out for in the logs/report_card?

**Comment by paoliniluis** [comment]
@BertrandToulas server logs don't matter here. It's a pure frontend issue where any value that is not recognized by the fe client will make the page reload.

So you archived question by question and still didn't work?

**Comment by BertrandToulas** [comment]
@paoliniluis Ah, I see. No, I haven't been able to comb through the questions on the frontend so far. I take it this is the only workaround then. Will try whenever possible. Hopefully it'll work. Thank you!

**Comment by marcelommm** [comment]
We are also experiencing the same issue in our organization.

@BertrandToulas Would you be kind to share how you found the culprits? All our jsons in report_card.dataset_query are valid, and we don't have any empty. We have +1800 questions, hence not possible to try one by one.

@flamber Can this be fixed in a future version?

**Comment by BertrandToulas** [comment]
@marcelommm sure thing! Like @paoliniluis explained to in a related issue, you want to create a user who has access to none of your Collections/questions, then create an empty collection just for them. Have the new user create a couple questions, a new dashboard, insert one of the questions into it, then try and add the other question via Add Series. For now Add Series should work because it will only look for the question inside the collection your user has access to (i.e. only the new collection).

After that, just use an Admin account to give the user access to another collection, switch back to the user and see if Add Series still works. If it does, add another collection, and another, until Add Series breaks. Once you found a collection that breaks Add Series, switch access to that collection's sub-folders one by one to zero in on the culprit(s). IMO the most convenient way to find the corrupted question(s) is to just "quarantine" a subset of questions (say, 10 of them) into the admin user's personal collection and see if Add Series is still broken on the test user's end. If it's working again, the culprit(s) is among those 10 questions you just put into quarantine. Move them back one-by-one, and check Add Series with the test user each time. AFAIK, corrupted question cannot just be fixed; you need to archive them.

Hopefully this makes sense. It should take you 2-3 hours, depending on how many questions happen to be corrupted.

**Comment by paoliniluis** [comment]
When https://github.com/metabase/metabase/pull/27058 is merged, the Add series modal should work and also spit out in the console the offending cards

**Comment by marcelommm** [comment]
Hi @flamber,

We've updated our MB instance to the latest today (0.45), but Add Series not working due to corrupted question persists.   

We spent a few hours trying to find culprit questions manually;   found a few ones,  but it's just too time consuming (we have 2100+ questions).

There isn't really a way for an automated fix?  If not we'll not be able to use Add Series ever again.

**Comment by paoliniluis** [comment]
Hi @marcelommm, we tried currupting cards on local instances and saw that the issue went away. It should now spit the corrupted cards on the browser console, do you see that?

**Comment by marcelommm** [comment]
Hi @paoliniluis, attached is the browser console.

**Comment by marcelommm** [comment]
Hi @paoliniluis, attached is the browser console.

![image](https://user-images.githubusercontent.com/112194436/213158090-11867d77-d912-431a-9e65-21e67228e95f.png)

Diagnostic info:
```
{
  "browser-info": {
    "language": "pt-PT",
    "platform": "Linux x86_64",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.17+8",
    "java.vendor": "Eclipse Adoptium",
    "java.vendor.url": "https://adoptium.net/",
    "java.version": "11.0.17",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.17+8",
    "os.name": "Linux",
    "os.version": "4.14.281-212.502.amzn2.x86_64",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "13.7"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.5.0"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2022-12-07",
      "tag": "v0.45.1",
      "branch": "release-x.45.x",
      "hash": "019d31c"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Comment by paoliniluis** [comment]
You’re using 45.1 @marcelommm , the fix was included in 45.2
